packages/synapse-react-client/src/utils/functions/RegularExpressions.ts

```diff
@@ -17,7 +18,7 @@ export const DOI_REGEX = /^10.\d{4,9}\/[-._;()/:a-z0-9]+$/i
  * > result[2]
  * '9'
  */
-export const SYNAPSE_ENTITY_ID_REGEX = /^(syn\d+)(?:\.(\d+))?$/
+export const SYNAPSE_ENTITY_ID_REGEX = /^(syn\d+)(?:\.(\d+))?$/i
```

Comment:
make this regex case insensitive, hoping that this will preemptively solve other issues that may be lurking